services/processor/native/get_code.go

```diff
@@ -90,7 +95,30 @@ func (s *service) retrieveDeployedContractInfoFromState(ctx context.Context, exe
 	return newContractInfo, nil
 }
 
-func (s *service) callGetCodeOfDeploymentSystemContract(ctx context.Context, executionContextId primitives.ExecutionContextId, contractName string) ([]byte, error) {
+func (s *service) getFullCodeOfDeploymentSystemContract(ctx context.Context, executionContextId primitives.ExecutionContextId, contractName string) ([][]byte, error) {
+	codeBytes, err := s.callGetCodeOfDeploymentSystemContract(ctx, executionContextId, contractName, 0)
```

Comment:
why this separation ?

Reply:
For backwards compatibility. So we can check that it worked for a singe case and then if we have more we will also fetch them.

Reply:
My goal here was to do small steps and not break the existing state of the `_Deployments` contract.